specs/PermissionManagement.md

```diff
@@ -379,6 +380,12 @@ typedef enum COREWEBVIEW2_PERMISSION_KIND {
   /// developers use the [Local Font Access API](https://wicg.github.io/local-font-access/)
   /// to query the system fonts available for styling web content.
   COREWEBVIEW2_PERMISSION_KIND_LOCAL_FONTS,
+
+  /// Indicates permission for a site to use system exlusive messages to access
+  /// Musical Instrument Digital Interface (MIDI) devices. Permission is requested
+  /// when developers use the [Web MIDI API](https://developer.mozilla.org/en-US/docs/Web/API/Web_MIDI_API)
+  /// to request system exclusive MIDI access.
```

Comment:
Does "Exclusive" mean granting this permission will interrupt or break a site that's already requested this permission?  That seems unique among the existing PermissionKind. Is it worth mentioning in documentation to the developers who will be using this enum?

Reply:
'sysex' is a MIDI term (apparently) that we are expanding here. Its in the DOM API as well: https://developer.mozilla.org/en-US/docs/Web/API/MIDIAccess/sysexEnabled. I think it would be good to elaborate in the documentation on what system exclusive is.

Reply:
Again, we should be careful not to say "system exclusive MIDI access" if this is really "access to system exclusive MIDI messages."